src/Sonarr.Api.V3/ImportLists/ImportListExclusionController.cs

```diff
@@ -29,9 +29,12 @@ protected override ImportListExclusionResource GetResourceById(int id)
 
         [HttpGet]
         [Produces("application/json")]
-        public List<ImportListExclusionResource> GetImportListExclusions()
+        public PagingResource<ImportListExclusionResource> GetImportListExclusions([FromQuery]PagingRequestResource paging)
```

Comment:
Since this changes the structure of the return it's a breaking change. Instead of breaking the API this should be added as a separate route and the old route can be deprecated.

`/paged` on the existing route should be sufficient.